agent/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# Install nsenter move to ubuntu:14.04.1
-FROM ibuildthecloud/ubuntu-core-base:14.04-1
+FROM ubuntu:14.04.1
```

Comment:
The idea is reproducibility.  I'm sure there's multiple opinions here, but the intentions was that if you built the image from the tag it should always yield the same result.